src/main/java/redis/embedded/AbstractRedisInstance.java

```diff
@@ -127,6 +127,8 @@ private void cancelStartupTimeoutCheck(ScheduledExecutorService executor) {
 
     protected abstract Pattern redisReadyPattern();
 
+    public abstract void setRedisReadyPat(Pattern redisReadyPat);
```

Comment:
I'd like to have this set via the `Builder` pattern, and not be able to mutate this class. Could you remove this method, and set the `Pattern` via the `Builder`s instead ?